source/access/managing_disk_usage.rst

```diff
@@ -1,105 +1,138 @@
-reaches this limit, no more information can be written to the 
-file system, and programs that try will fail.                    
-                                  
-Some file systems have limits on the number of files that can be   
-stored, and those are represented by the last four columns. The     
-number of files you currently have is listed in the column      
-files, quota and limit represent  the soft and hard limits for the  
-number of files.                  
-                                  
-Diskspace used by individual directories         
-----------------------------------------
-                                  
-The command to check the size of  all subdirectories in the current 
-directory is ``du``
-                                  
-::                                
-                                  
-   $ du -h                        
-   4.0k      ./.ssh               
-   0         ./somedata/somesubdir  
-   52.0k     ./somedata             
-   56.0k   .                      
-                                  
-                                  
+                 648M   2919M   3072M            3685       0       0
+     nas2-ib1:/mnt/data
+               20691M  24320M  25600M            134k       0       0
+     nas1-ib1:/mnt/site_scratch
+                    0  24320M  25600M               1       0       0
+
+Each line represents a file system you have access to, ``$VSC_HOME``,
+``$VSC_DATA``, and, for this particular example, ``$VSC_SCRATCH_SITE``.
+When using the ``-s`` flag, ``quota`` will report disk space and limits
+in human-readable format, i.e., using MB or GB, rather than blocks.
+
+Some file systems have limits on the number of files that can be
+stored, and those are represented by the last four columns. The
+
+``files``
+   The number of files on the file system owned by you.
+``quota``
+   If the number of files exceeds this value, you may
+   get warnings (disabled on most systems).  The reported number
+   of files will be marked with an ``*`` if it exceeds the value
+   of ``quota``.
+``limit``
+   If the limit is reached, no new files can be written to disk, so
+   applications and jobs may crash.
+
+.. note::
+
+   Using these commands on another cluster than the one
+   in your home institution, will fail to return information
+   about the quota on your ``$VSC_HOME`` and ``$VSC_DATA``
+   directories and will show you  the quota for your ``$VSC_SCRATCH``
+   directory on that system.
+
+
+.. _mmlsquota:
+
+Total disk space on GPFS file systems
+-------------------------------------
+
+On some parallel file systems, notably ``$VSC_SCRATCH`` on KU Leuven
+infrastructure (Tier-1 and Tier-2), ``mmlsquota`` should be used to
+display the total disk space you use and the limits you have.
+
+These file system is a parallel file system, i.e., IBM Spectrum Scale (formerly
+GPFS)::
+
+      $ mmlsquota  --block-size auto  vol_ddn2:leuven_scratch
```

Comment:
Next to Stijn his remarks I also don't think it's a good idea to give these kinds of command to the users. At the minimum a wrapper that hides all the details should be used.